doc/developer.md

```diff
@@ -22,21 +22,24 @@ We recommend this for development work ONLY.
     pip install setuptools --upgrade
     ```
     
-3. Install requirements.txt
-    ```
-    pip install -r requirements.txt
-    ```
-
-4. Clone the repo to `/var/www/html`
+3. Clone the repo to `/var/www/html
```

Comment:
This doesn't seem wise. Won't this make every file be served by Apache? Even things that aren't meant to be?

Reply:
It is probably not wise, see main comment